lib/amqp.js

```diff
@@ -145,16 +144,34 @@ AMQPConnection.prototype.sendToExchange = function sendToExchange(exchangeName,
     }
 };
 
+AMQPConnection.prototype.prepareMessageAndSendToExchange
```

Comment:
AMQPConnection could be written in es2015 style, so that this string becomes much shorter